tiledb/sm/crypto/test/unit_crypto.cc

```diff
@@ -35,10 +35,34 @@
 #include "tiledb/sm/crypto/crypto.h"
 
 #include <test/support/tdb_catch.h>
+#include <iomanip>
 #include <iostream>
 
 using namespace tiledb::sm;
 
+#ifdef _WIN32
+#include "tiledb/sm/crypto/crypto_win32.h"
+using PlatformCrypto = Win32CNG;
+#else
+#include "tiledb/sm/crypto/crypto_openssl.h"
+using PlatformCrypto = OpenSSL;
+#endif
```

Comment:
I'm thinking for a future PR to merge `Win32CNG` and `OpenSSL` into two possible implementations of `Crypto` depending on the platform.

For now another thing I thought is to expose a `get_random_bytes` method in the `Crypto` class, but it would be used only for tests, if that is a problem.

Reply:
Done.